examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,250 @@
+a common reference space. It will be demonstrated using the SourceMorp class.
+The example uses parts of the MNE example
+:ref:`sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py` and
+:ref:`sphx_glr_auto_examples_inverse_plot_mne_dspm_source_localization.py`.
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain) in VolSourceEstimate
+case and using an affine transform in the SourceEstimate or
+VectorSourceEstimate case. Afterwards the transformation will be applied to the
+beamformer result. The result will be a plot showing the morphed result
+overlaying the fsaverage T1. Uncomment at the respective location to plot the
+result of the surface morph.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pylab as plt
+import mne
+import nibabel  as nib
+import numpy as np
+from mne import SourceMorph
+from mne.beamformer import make_lcmv, apply_lcmv
+from mne.minimum_norm import make_inverse_operator, apply_inverse
+from nilearn.plotting import plot_anat
+
+print(__doc__)
+
+
+###############################################################################
+# from :ref:`sphx_glr_auto_examples_inverse_plot_mne_dspm_source_localization.py`.
+# Note, that in order to save the data fname needs to be set
+def compute_lcmv_example_data_surf(data_path, fname=None):
+    raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+
+    raw = mne.io.read_raw_fif(raw_fname)  # already has an average reference
+    events = mne.find_events(raw, stim_channel='STI 014')
+
+    event_id = dict(aud_r=1)  # event trigger and conditions
+    tmin = -0.2  # start of each epoch (200ms before the trigger)
+    tmax = 0.2  # end of each epoch (500ms after the trigger)
+    raw.info['bads'] = ['MEG 2443', 'EEG 053']
+    picks = mne.pick_types(raw.info, meg=True, eeg=False, eog=True,
+                           exclude='bads')
+    baseline = (None, 0)  # means from the first instant to t = 0
+    reject = dict(grad=4000e-13, mag=4e-12, eog=150e-6)
+
+    epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
+                        picks=picks,
+                        baseline=baseline, reject=reject)
+    noise_cov = mne.compute_covariance(
+        epochs, tmax=0., method=['shrunk', 'empirical'], verbose=True)
+
+    evoked = epochs.average().pick_types(meg=True)
+
+    del epochs  # to save memory
+
+    # Read the forward solution and compute the inverse operator
+    fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-oct-6-fwd.fif'
+    fwd = mne.read_forward_solution(fname_fwd)
+
+    # make an MEG inverse operator
+    info = evoked.info
+    inverse_operator = make_inverse_operator(info, fwd, noise_cov,
+                                             loose=0.2, depth=0.8)
+
+    method = "dSPM"
```

Comment:
wait, the name of the function is `compute_lcmv_example_data_surf` ...

what am I missing here? and there shouldn't be a function anyway